modules/sdk-coin-sol/src/sol.ts

```diff
@@ -415,9 +415,13 @@ export class Sol extends BaseCoin {
     let rebuiltTransaction;
 
     try {
-      const transactionBuilder = factory.from(params.txBase64).fee({ amount: params.feeInfo.fee });
-      if (params.tokenAccountRentExemptAmount) {
-        transactionBuilder.associatedTokenAccountRent(params.tokenAccountRentExemptAmount);
+      const transactionBuilder = factory.from(params.txBase64);
+      if (transactionBuilder instanceof TransactionBuilder) {
+        const txBuilder = transactionBuilder as TransactionBuilder;
```

Comment:
Do you still need the case here if it's inside the `instanceof` if statement?

Reply:
yeah, compiler still complain if not.